### PR TITLE
Connect data to API

### PR DESCRIPTION
**Pull request by GabrielRuizVarela**

### CODE REVIEW

**Review by thom-as-thom — APPROVED** [review]

File: src/containers/CryptoList/CryptoList.tsx [hunk 1]
```diff
@@ -10,55 +11,32 @@ import {
...
-);
+const CryptoList = () => {
+  const {cryptos: fetchCrypto} = useAppSelector(state => state);
+  const dispatch = useAppDispatch();
+  const AddCrypto = () => {
+    dispatch(fetchCryptoWithSymbol('btc'));
+    dispatch(fetchCryptoWithSymbol('eth'));
+    dispatch(fetchCryptoWithSymbol('xrp'));
+  };
+  return (
```
Comment (VicenBayugar) [inline src/containers/CryptoList/CryptoList.tsx]: leave a blank space before the return

File: src/types/png.d.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+declare module '*.png' {
+  const content: any;
```
Comment (VicenBayugar) [inline src/types/png.d.ts]: don't use any type

Comment (GabrielRuizVarela) [reply src/types/png.d.ts]: Vincen, que type debería usar en este caso?

File: src/containers/CryptoList/index.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  AvatarImg,
+  Title,
+  TopBar,
+  TouchableAddCrypto,
+} from './styles';
+import React from 'react';
+import {FlatList, View} from 'react-native';
+
+const CryptoList = () => {
+  const {cryptos: fetchCrypto} = useAppSelector(state => state);
```
Comment (FedericoLeiva12) [inline src/containers/CryptoList/index.tsx]: Maybe is better to do the destructuring inside the useAppSelector

useAppSelector(({ cryptos }) => cryptos)

**Pull request merged**